core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -30,6 +31,8 @@
 
     public static final String SMART_TESTING = "smart.testing";
     public static final String SMART_TESTING_MODE = "smart.testing.mode";
+    public static final String SMART_TESTING_CUSTOM_STRATEGIES = "smart.testing.strategy";
```

Comment:
I know we haven't used `smart.testing.strategy` variable/property previously. how about `smart.testing.custom.strategy` ?

It won't be confusing for user as well.

Reply:
Yes we used previously in previous versions when you could register strategies using system properties.

Reply:
no we are using `smart.testing` for previous versions. We haven't used `smart.testing.strategies`. 
https://github.com/arquillian/smart-testing/pull/212/files/217a5df9249ef1e56eaba84be0c264ee13a49eed#diff-348861c21efd28f5cc99117b2c67302aR32

Reply:
It seems because I moved the bits but look at current master https://github.com/arquillian/smart-testing/blob/master/mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/StrategyDependencyResolver.java#L29

Reply:
wdyt about `smart.testing.custom.strategies`?

Reply:
@bartoszmajsak any opinion about this? In my opinion original attribute name was ok, since it gives "custom" strategies the same level as internal strategies, but of course I am open to change.